kolibri/plugins/management/assets/src/device_management/state/actions/manageContentActions.js

```diff
@@ -14,8 +14,8 @@ export function refreshChannelList(store) {
 }
 
 export function showManageContentPage(store) {
+  store.dispatch('RESET_MANAGE_CONTENT_PAGESTATE');
```

Comment:
There was an error where the tasklist did not exist when navigating from the permissions to content tab. This resets the state regardless of permissions/signed in. @jonboiser